tests/test_spag.py

```diff
@@ -263,33 +224,31 @@ def test_spag_request_patch(self):
     def test_spag_request_delete(self):
         for name in ('delete_thing.yml', 'delete_thing'):
             _, _, ret = run_spag('post', '/things', '--data', '{"id": "a"}',
-                                  '-H', 'content-type: application/json')
+                                  '-H', 'content-type=application/json')
             self.assertEqual(ret, 0)
 
-            out, err, ret = run_spag('request', name, '--dir', RESOURCES_DIR)
-            self.assertEqual(ret, 0)
-            self.assertEqual(out, '\n')
-            self.assertEqual(err, '')
+        out, err, ret = run_spag('request', 'delete_thing.yml')
+        self.assertEqual(ret, 0)
+        self.assertEqual(out, '\n')
+        self.assertEqual(err, '')
```

Comment:
This last block either needs to be inside the loop (with `name` instead of `delete_thing.yml`) or only make one post request.